overlord/devicestate/firstboot.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/snapcore/snapd/asserts/snapasserts"
 	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/i18n"
+	"github.com/snapcore/snapd/image"
```

Comment:
I don't think we want to import in this direction though

Reply:
Moved CheckBasesAndProviders to snap package (and renamed).